blog/migrations/0003_stori_featured.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1.3 on 2024-03-06 15:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("blog", "0002_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="stori",
+            name="featured",
+            field=models.BooleanField(default=False),
+        ),
+    ]
```

Comment:
that a database migration is created and applied to reflect the new field in the Stori model.